### PR TITLE
Formatted floormod

### PR DESCRIPTION
**Pull request by kylejcharlton**
[#2082](https://github.com/unifyai/ivy/issues/2082)

### CODE REVIEW

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @kylejcharlton 
Thanks for the great work! There are minor changes required to the docstring, pleae check the comments. You can re-request review once it's done and I'll merge it

**Comment by kylejcharlton** [comment]
Hey @rush2406, I went ahead and made those changes to better reflect the container implementation. Let me know if everything looks good :)

File: ivy/array/general.py [hunk 1]
```diff
@@ -52,6 +52,55 @@ def all_equal(self: ivy.Array, x2: Iterable[Any], equality_matrix: bool = False)
...
+        --------
+        >>> x = ivy.array([9, 8, 7])
+        >>> y = ivy.array([2, 3, 4])
+        >>> print(x.floormod(y))
+        ivy.array([1, 2, 3])
+
+        >>> a = ivy.array([3, 5, 8])
+        >>> b = ivy.array([1, 2, 3])
+        >>> c = ivy.array([0, 0, 0])
+        >>> a.floormod(b, c)
```
Comment (rush2406) [inline ivy/array/general.py:91]: this should be `a.floormod(b,out=c)`. The keyword `out` is required otherwise the doctring tests are failing.

**Review by rush2406 — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/general.py [hunk 2]
```diff
@@ -1203,6 +1203,66 @@ def floormod(
...
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([9, 8, 7])
+    >>> y = ivy.native_array([2, 3, 4])
+    >>> print(ivy.floormod(x, y))
+    ivy.array([1, 2, 3])
+
+    >>> a = ivy.native_array([3, 5, 8])
+    >>> b = ivy.native_array([1, 2, 3])
+    >>> c = ivy.native_array([0, 0, 0])
```
Comment (rush2406) [inline ivy/functional/ivy/general.py:1237]: `out` is not supported with `ivy.native_array`, so ideally you can remove this example

**Review by rush2406 — CHANGES_REQUESTED** [review]

**Comment by kylejcharlton** [comment]
Hey @rush2406, I made those changes that you suggested and it appears that I am now passing the linting tests. Let me know if there's anything else for me to take care of :)

File: ivy/functional/ivy/general.py [hunk 3]
```diff
@@ -1203,6 +1203,53 @@ def floormod(
...
+    >>> y = ivy.Container(a = ivy.array([2, 3, 4]), b = ivy.array([1, 2, 3]))
+    >>> print(ivy.floormod(x, y))
+    {
+        a: ivy.array([1, 2, 3]),
+        b: ivy.array([0, 1, 2])
+    }
+
+    >>> a = ivy.Container(a = ivy.array([7, 6, 4]))
+    >>> b = ivy.array([4, 5, 4])
+    >>> ivy.floormod(a, b, out=b)
```
Comment (rush2406) [inline ivy/functional/ivy/general.py:1247]: here the `out` argument should of type container, so please change it to `out=a`

**Review by rush2406 — CHANGES_REQUESTED** [review]

**Comment by rush2406** [comment]
@kylejcharlton thanks for doing the changes, can you please check the `test-docstrings / run-docstring-tests` in this PR. It seems to be failing for a few examples

**Pull request closed without merging**